phoenix-core/src/test/java/org/apache/phoenix/cache/ServerMetadataCacheImplTest.java

```diff
@@ -76,11 +78,13 @@
 import static org.mockito.Mockito.verify;
 
 @Category(ParallelStatsDisabledIT.class)
-public class ServerMetadataCacheTest extends ParallelStatsDisabledIT {
+public class ServerMetadataCacheImplTest extends ParallelStatsDisabledIT {
```

Comment:
Do we want to rename this class to `ServerMetadataCacheIT` to avoid confusion between `ServerMetadataCacheImplTest` and `ServerMetadataCacheTestImpl`